docs/directives/columns-format-case.md

```diff
@@ -0,0 +1,53 @@
+# Converts the Case of Columns
+
+COLUMNS-FORMAT-CASE directive provides the ability to clean up the column names in bulk.
+
+## Syntax
+
+```
+ columns-format-case current-format desired-format
+```
+
+```current-format``` or ```desired-format``` specifies the current or desired format and can accept any of the following options:
```

Comment:
Why do we need to specify current-format ? Why not just tell what you need it to be. I found it confusing to understand what my current-format is.